apps/explorer/src/app/components/txs/details/tx-liquidity-amend.tsx

```diff
@@ -34,6 +34,8 @@ export const TxDetailsLiquidityAmendment = ({
     txData.command.liquidityProvisionAmendment;
   const marketId: string = amendment.marketId || '-';
 
+  const fee = amendment.fee ? parseFloat(amendment.fee) * 100 : '-';
```

Comment:
Suggestion (blocking): multiplying any rounding issues introduced by parseFloat by 100 could result in a fair amount of inaccuracy here. Using Bignumber would be a better approach.